LedgerSMB/Scripts/contact.pm

```diff
@@ -340,7 +341,7 @@ sub _main_screen {
                    person => $person,
                  employee => $person,
                      user => $user,
-                    roles => eval { $user->list_roles },
```

Comment:
should be roles =>  [ eval { $user->list_roles } ].

But your changes work too.